app/assets/stylesheets/live/layout.sass

```diff
@@ -44,15 +44,26 @@ body
   margin-top: 2%
   
 .timer__column__container
-  display: flex
+  //display: flex
 
 .timer__left
-  flex: 1
+  //flex: 1
 
 .timer__right
-  flex: 1
+  //flex: 1
 
 .timer__container
+  padding: 25px 0px 20px 0px
+
+#timer_submission
+  font-size: 125px
+  padding: 25px 0px 25px 0
+#submission_text
+  font-size: 40px
+  color: #F98728
+#timer_hack
+  font-size: 125px
+  padding: 25px 0px 25px 0px
```

Comment:
`#submission_text` should change to a class (e.g. `.submission_text`) so you aren't using the same ID multiple times on the page

Also maybe want to consolidate `#timer_submission` and `#timer_hack` into one class since they have similar styles

Reply:
Another ping on this